libraries/testkit/sgaccel.py

```diff
@@ -39,16 +41,24 @@ def stop(self):
 
     def start(self, config):
         conf_path = os.path.abspath(config)
-
         log.info(">>> Starting sg_accel with configuration: {}".format(conf_path))
 
+        playbook_vars = {
+            "sync_gateway_config_filepath": conf_path,
+            "server_port": self.server_port,
+            "server_scheme": self.server_scheme
+        }
+        if is_cbs_ssl_enabled(self.cluster_config) and get_sg_version(self.cluster_config) >= "1.5.0":
```

Comment:
get_sg_version is returning two variables, are we comparing self._version_number to "1.5.0"?

Reply:
yes, get_sg_version returns only one variable which is what i have added. Btw, let me show you tomorrow that it can work without calling comparing versions